test/Microsoft.AspNetCore.Mvc.Razor.Test/RazorViewEngineTest.cs

```diff
@@ -1710,18 +1677,26 @@ private RazorViewEngine CreateSuccessfulViewEngine()
 
         private TestableRazorViewEngine CreateViewEngine(
             IRazorPageFactoryProvider pageFactory = null,
-            IEnumerable<IViewLocationExpander> expanders = null)
+            IEnumerable<IViewLocationExpander> expanders = null,
+            IEnumerable<string> viewLocationFormats = null,
+            IEnumerable<string> areaViewLocationFormats = null)
         {
             pageFactory = pageFactory ?? Mock.Of<IRazorPageFactoryProvider>();
             return new TestableRazorViewEngine(
                 pageFactory,
-                GetOptionsAccessor(expanders));
+                GetOptions(expanders, viewLocationFormats, areaViewLocationFormats));
         }
 
-        private static IOptions<RazorViewEngineOptions> GetOptionsAccessor(
-            IEnumerable<IViewLocationExpander> expanders = null)
+        private static IOptions<RazorViewEngineOptions> GetOptions(
+            IEnumerable<IViewLocationExpander> expanders = null,
+            IEnumerable<string> viewLocationFormats = null,
+            IEnumerable<string> areaViewLocationFormats = null)
         {
+            var optionsSetup = new RazorViewEngineOptionsSetup(Mock.Of<IHostingEnvironment>());
```

Comment:
As I said, it's weird to use (and then override) the `RazorViewEngineOptionsSetup` instance in a test of `RazorViewEngine`. These are unit tests and we should be careful about adding unnecessary dependencies.